mikochiku_alarm.py

```diff
@@ -117,23 +118,27 @@ def set_target_channel(self, qmode8ndex):
 
     def check_live(self):
         buff_video_id_set = self.get_live_video_id(self.search_ch_id)
-        for getting_video_id in buff_video_id_set:
+        for getting_video_id in buff_video_id_set.keys():
             if getting_video_id in self.old_video_id_list:
-                return
+                continue
             self.old_video_id_list.append(getting_video_id)
             if len(self.old_video_id_list) > 30:
                 self.old_video_id_list.pop(0)
             log.info(self.localized_text("started"))
-            log.debug(f"buff_video_id_set: {buff_video_id_set}")
+            log.debug(f"buff_video_id_set: {[id for id in buff_video_id_set.keys()]}")
             log.debug(f"self.old_video_id_list {self.old_video_id_list}")
             self.alarm_stop.click()
             self.alarm_state = "stop"
             self.alarm_stop.setText(self.localized_text("stop"))
+            opened = False
             if self.webbrowser_cb.checkState():
                 webbrowser.open(
                     "https://www.youtube.com/watch?v=" + getting_video_id)
+                opened = True
             if self.alarm_cb.checkState():
                 self.alarm_sound()
+            t = toast.Toast(self, getting_video_id, buff_video_id_set[getting_video_id], opened)

```

Comment:
```suggestion
            should_open_browser = self.webbrowser_cb.checkState()
            if should_open_browser:
                webbrowser.open(
                    "https://www.youtube.com/watch?v=" + getting_video_id)
            if self.alarm_cb.checkState():
                self.alarm_sound()
            t = toast.Toast(self, getting_video_id, buff_video_id_set[getting_video_id], should_open_browser)
```

`self.webbrowser_cb.checkState()` can be reused.

Be aware that `should_open_browser == True` means 'should NOT open browser' in the toast.

Reply:
Fixed https://github.com/pusaitou/mikochiku_alarm/pull/61/commits/2bc88200f193221dce2107777123e83bda8620c3